README.md

```diff
@@ -146,6 +146,8 @@ decrypted content does not fit the schema displayed below.**
 | fieldType   | string   | The type of field for the question.                                                                      |
 | \_id        | string   | A unique identifier of the form field. WARNING: Changes when new fields are created/removed in the form. |
 
+Additional internal fields may be included from time to time, which will then be published above once it is stable for public consumption.
```

Comment:
```suggestion
**Important Note: **
Additional internal fields may be included in webhooks from time to time, which will then be published as part of our official schema once it is stable for public consumption. If you are applying your own validation, you should account for this e.g. by not rejecting the webhook if there are additional fields included.
```

Reply:
hehe @KenLSM see if this is ok!

Reply:
Haha iirc other languages (like golang) will automatically drop unspecified schemas